src/phoenix/session/session.py

```diff
@@ -174,6 +177,7 @@ def __init__(
         reference_dataset: Optional[Dataset] = None,
         corpus_dataset: Optional[Dataset] = None,
         trace_dataset: Optional[TraceDataset] = None,
+        default_umap_parameters: Optional[Dict[str, Union[int, float]]] = None,
```

Comment:
```suggestion
        default_umap_parameters: Optional[Mapping[str, Any]] = None,
```

Reply:
Agree that the type is incorrect - though the former was a bit less error prone - will switch to any for now